OpenRA.Game/Activities/Activity.cs

```diff
@@ -12,12 +12,29 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using OpenRA.Primitives;
 using OpenRA.Traits;
 
 namespace OpenRA.Activities
 {
 	public enum ActivityState { Queued, Active, Canceling, Done }
 
+	public struct TargetLineNode
```

Comment:
Not sure if this should be a struct, its size is 88 bytes.  See https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct

Reply:
Converted to class.